### PR TITLE
Backport fix for #766834

### PR DESCRIPTION
**Pull request by jschueller**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -53,10 +55,12 @@ test:
...
   license: MIT
+  license_file: Copyright
   summary: 'The XML C parser and toolkit of Gnome.'
 
 extra:
   recipe-maintainers:
     - ocefpaf
     - jakirkham
     - gillins
+    - jschueller
```
Comment (ocefpaf) [inline recipe/meta.yaml]: :+1:

**Comment by ocefpaf** [comment]
@tomkralidis, @rsignell-usgs, and @kwilcox this patch should allow us to use `libxml2 2.9.4` in `pycsw`.

PS: But I will unpin `pycsw` until we perform some tests. Do no worry.

**Comment by tomkralidis** [comment]
cc @kalxas 

Thanks @ocefpaf very much for tracking this down.  I've shared this with the lxml mailing list for info as well.

**Comment by ocefpaf** [comment]
Thanks @jschueller :wink: (I took the lazy path to pin it to the version that worked. @jschueller tracked this down.)

**Pull request merged**